simplq/src/components/pages/Home/MyQueues.jsx

```diff
@@ -4,15 +4,18 @@ import IconButton from '@material-ui/core/IconButton';
 import { useHistory } from 'react-router';
 import { useSelector } from 'react-redux';
 import styles from './home.module.scss';
-import { deleteQueue } from '../../../services/queue';
+import { QueueRequestFactory } from '../../../api/requestFactory';
+import useRequest from '../../../api/useRequest';
 
 export default () => {
   const history = useHistory();
   const myQueues = useSelector((state) => state.appReducer.myQueues);
+  const { requestMaker } = useRequest();
+
   const handleDelete = (e, queue) => {
     // Don't trigger parent's onClick
     e.stopPropagation();
-    deleteQueue(queue.queueId).then(() => history.push('/'));
+    requestMaker(QueueRequestFactory.deleteQueue(queue.queueId)).then(() => history.push('/'));
   };
```

Comment:
deleteQueue used to handle modifying the global store too. Is that being done in this version?
`await getMyQueues().then((subResponse) => store.dispatch(setMyQueues(subResponse.queues)));`

Reply:
This was breaking our pattern, and coupling stuff. That was a bad design choice from my side. I am undoing it in next PR.

In the component where we require this data, now we can just call getMyQueues when displaying queues to the user. (Straigtforward like any other API call, no redux, no setting it here an there)